include/zephyr/dt-bindings/gpio/realtek-gpio.h

```diff
@@ -39,4 +39,112 @@
 /** Interrupt Mask since rts5912 does not support GPIO_INT_LEVELS_LOGICAL*/
 #define RTS5912_GPIO_INTR_MASK   (1U << 21 | 1U << 22 | 1U << 24 | 1U << 25 | 1U << 26)
 
+/** GPIO number map between spec and zephyr */
```

Comment:
Non-blocking suggestion "Map GPIO signal name to devicetree binding."

Please also add a comment describing how to use this macro.  See dt-bindings/gpio/microchip-xec-gpio.h for an example.